test/uk/gov/hmrc/excisemovementcontrolsystemapi/connectors/EISSubmissionConnectorSpec.scala

```diff
@@ -132,94 +139,85 @@ class EISSubmissionConnectorSpec
       clean(captor.value) mustBe clean(controlWrappedXml.toString)
     }
 
-    "return Bad request error" in {
+    "return error when a bad request response returned" in {
 
       when(mockHttpClient.post(any)(any)).thenReturn(mockRequestBuilder)
       when(mockRequestBuilder.setHeader(any())).thenReturn(mockRequestBuilder)
       when(mockRequestBuilder.withBody(any())(any(), any(), any())).thenReturn(mockRequestBuilder)
-      when(mockRequestBuilder.execute[Either[Result, EISSubmissionResponse]](any(), any()))
-        .thenReturn(Future.successful(Left(BadRequest("any error"))))
+      when(mockResponse.json).thenReturn(Json.toJson(EISErrorResponse(timestamp, "400", "", "", "")))
+      when(mockResponse.status).thenReturn(BAD_REQUEST)
 
       val result = await(submitExciseMovementForIE815)
-      result.left.value mustBe BadRequest("any error")
+      result.left.value mustBe EISErrorResponseDetails(BAD_REQUEST, timestamp, "", "", "", None)
     }
 
-    "return 500 if post request fail" in {
+    //Should this instead be a Future.failed?
```

Comment:
is exploration of this outstanding before merge?

Reply:
I'd say not in this ticket, technical debt probably.